### DISCUSSION TITLE
Upgrading to PHP8

### DISCUSSION THREAD

**Discussion by InduBhusanNath** [leokhoa/laragon#542] [Q&A]
I have upgraded Laragon to PHP8 version. But when I type php -v, it show PHP7

**Comment by Pen-y-Fan** [comment]
If you use Laragon's terminal (cmdr) the paths are automatically updated, if you use a different terminal you will need to add Laragon to Windows PATHS, see my blog [How to set up Laragon on a new Windows computer (part 6) - Add to path](https://pen-y-fan.github.io/2022/11/08/how-to-set-up-laragon-on-a-new-windows-computer-part-2/)

**Reply by InduBhusanNath** [reply]
You need to download it and install it. To get the path, you can click on the Terminal button. And the rest is self-explanatory.

**Reply by InduBhusanNath** [reply]
Great!

**Comment by luisAntonioLAGS** [comment]
Remove and re-add laragon to the environment variables using the path tool in the menu. Sometimes it is necessary to restart the system for it to work.

![imagen](https://github.com/leokhoa/laragon/assets/60680854/99793463-2c52-4997-ae2c-7a7cbadf6f51)
